Libraries/Animated/src/NativeAnimatedModule.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright 2004-present Facebook. All Rights Reserved.
+ *
+ * @flow strict-local
+ * @format
+ */
+
+'use strict';
+
+import type {TurboModule} from 'RCTExport';
+import * as TurboModuleRegistry from 'TurboModuleRegistry';
+import type {EndCallback} from './animations/Animation';
+
+export type EventMapping = {|
+  nativeEventPath: Array<string>,
+  animatedValueTag: ?number,
+|};
+
+export type AnimatedNodeConfig = {|
+  type?:
+    | 'style'
+    | 'value'
+    | 'props'
+    | 'interpolation'
+    | 'addition'
+    | 'diffclamp'
+    | 'division'
+    | 'multiplication'
+    | 'modulus'
+    | 'subtraction'
+    | 'transform'
+    | 'tracking',
+|};
+
+export type AnimatingNodeConfig = {|
+  type?: 'frames' | 'spring' | 'decay',
+|};
+
+export interface Spec extends TurboModule {
+  +createAnimatedNode: (tag: ?number, config: AnimatedNodeConfig) => void;
+  +startListeningToAnimatedNodeValue: (tag: ?number) => void;
+  +stopListeningToAnimatedNodeValue: (tag: ?number) => void;
+  +connectAnimatedNodes: (parentTag: ?number, childTag: ?number) => void;
+  +disconnectAnimatedNodes: (parentTag: ?number, childTag: ?number) => void;
+  +startAnimatingNode: (
+    animationId: ?number,
+    nodeTag: ?number,
+    config: AnimatingNodeConfig,
+    endCallback: EndCallback,
```

Comment:
type imported from other files doesn't work well with our codegen atm, so just inline the type here or copy the type over in this file